packages/style/components/queryBuilder/QueryBar.module.scss

```diff
@@ -65,6 +67,15 @@
             >  :first-child {
                 margin-right: .5rem;
             }
+
+            img,svg {
+                height: 22px;
```

Comment:
you cant hardcod height of the icons in CQDG it is 18px. And not sure why we put this css here, you have full control of the icon pass down to Querybuilder, you can add a classname and style it outside of the queryBuilder

Reply:
my bad here, fixing it